MarkBind/markbind
# Change Panel documentation in 'dynamic' sub-section to 'static'

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Documentation update


Resolves #326 



**What is the rationale for this request?**
'dynamic' attribute for Panel has been changed to 'static', documentation requires an update.

**What changes did you make? (Give an overview)**
- Change 'dynamic' sub-section to 'static'
- Rename 'dynamic/' folder to 'anotherSrc'

**Is there anything you'd like reviewers to focus on?**
- Wording of new sub-section

**Testing instructions:**
1. Get `vue-strap.min.js` from building MarkBind/vue-strap#67 
2. Run `markbind serve docs` and check the new section.
